test/uk/gov/hmrc/uknwauthcheckerapi/controllers/AuthorisationsControllerSpec.scala

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmrc.uknwauthcheckerapi.controllers
+
+import org.apache.pekko.actor.setup.Setup
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.libs.json.Json
+import play.api.mvc.Result
+import play.api.test.Helpers._
+import play.api.test.{FakeRequest, Helpers}
+import uk.gov.hmrc.uknwauthcheckerapi.models.{AuthorisationRequest, AuthorisationsResponse, EISResult}
+
+import scala.concurrent.Future
+
+class AuthorisationsControllerSpec extends AnyWordSpec with Matchers {
+
+  private val controller = new AuthorisationsController(Helpers.stubControllerComponents())
+
+  "AuthorisationsController" should {
+
+    val converted =
+      AuthorisationsResponse(
+        "01-01-1999",
```

Comment:
Use a LocalDate generator.

Reply:
Introduced generator for localDate and eori